bin/yb-ctl

```diff
@@ -646,14 +646,15 @@ class ClusterOptions:
             self.use_cassandra_authentication = args.use_cassandra_authentication
 
         if hasattr(args, "ip_start"):
-            self.ip_start = args.ip_start
-
-        if hasattr(args, "listen_ip"):
-            self.listen_ip = args.listen_ip
-            if (self.replication_factor == 1 and not self.listen_ip):
-                self.listen_ip = '0.0.0.0'
-            if self.listen_ip and self.replication_factor > 1:
-                raise RuntimeError("Invalid argument: listen_ip is only compatible with rf=1")
+            if args.ip_start is not None:
+                self.ip_start = args.ip_start
+            elif hasattr(args, "listen_ip"):
+                # Only set listen_ip if ip_start is not given.
+                self.listen_ip = args.listen_ip
+                if (self.replication_factor == 1 and not self.listen_ip):
```

Comment:
i guess you wanted to make this `and not self.ip_start`

Reply:
`self.ip_start` is always truthy.  This is because it gets initialized as `DEFAULT_IP_START` (`1`).  The only way to distinguish `ip_start` being unspecified is through `args.ip_start`, and I do just that with the `if` and `elif` blocks.